platforms/corda-ent/charts/nmap/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v2
+appVersion: "1.2"
+description: A Helm chart for CENM Networkmap
+name: nmap
+version: "1.0.0"
```

Comment:
newline needed where there is this red sign

Reply:
Changed in commit [d778500](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/841/commits/d77850089dd8c95b11ae4d8e96b42480fd82c838)